src/lib/web3-utils.js

```diff
@@ -196,3 +196,18 @@ export async function checkValidEthNode(uri) {
 
   return true
 }
+
+export async function signMessage(wallet, message) {
+  let signHash
+  let error = false
+  let errorMsg
+
+  try {
+    signHash = await wallet.ethers.getSigner().signMessage(message)
+  } catch (err) {
+    error = true
+    errorMsg = err
```

Comment:
no because we are not showing on the designs the error but given that we have severals possible errros at some point we would want to show the error

Reply:
In that case, does it make sense to modify `error` to be the error message and not a boolean, and also drop `errorMsg`? I'm thinking this because looks like we can indicate whether there was en error or not directly from the error returned value. Returning both `error` and `errorMsg` feels like having duplicated logic.